features/step_definitions/admin_form_steps.rb

```diff
@@ -1,22 +1,11 @@
-When(/^I visit the auctions admin page$/) do
```

Comment:
should this file me named `admin_auction_form_steps`? Not sure but seems like this is all related to the auction form. worried that `admin_form` is too vague, although we only have one form right now . Or do we?